### PR TITLE
add docker-compose & add storage

### PR DESCRIPTION
**Pull request by xander42280**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # zkm-prover
-A parallel proving service for [ZKM](https://github.com/zkMIPS/zkm).
+A parallel proving network for [ZKM](https://github.com/zkMIPS/zkm).
```
Comment (eigmax) [inline README.md:2]: No need to change it back.

File: prover/src/pipeline.rs [hunk 2]
```diff
@@ -10,7 +10,7 @@ pub struct Pipeline {
     _mutex: Mutex<usize>,
 }
 
-static PIPELINE_MUTEX: Mutex<usize> = Mutex::new(0);
+pub static PIPELINE_MUTEX: Mutex<usize> = Mutex::new(0);
```
Comment (eigmax) [inline prover/src/pipeline.rs:13]: We should not expose this to public?

File: prover/src/provers/agg_all_prover.rs [hunk 3]
```diff
@@ -50,6 +50,10 @@ impl Prover<AggAllContext> for AggAllProver {
             return Ok(());
         }
 
+        if proof_num < 1 {
```
Comment (eigmax) [inline prover/src/provers/agg_all_prover.rs:53]: This is exact as same as L49-51.

File: service/src/storage.rs [hunk 4]
```diff
@@ -0,0 +1,74 @@
+use mysql_async::prelude::*;
```
Comment (eigmax) [inline service/src/storage.rs:1]: I think we should use sqlx to support different storage backends, and the sqlx also provides a friendly way to track database changes.  https://docs.rs/sqlx/latest/sqlx/macro.migrate.html

Comment (xander42280) [reply service/src/storage.rs]: ok

**Pull request closed without merging**